app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -94,9 +95,26 @@
     this.setOrganisationsDimension(dimensions['analytics:organisations']);
     this.setWorldLocationsDimension(dimensions['analytics:world-locations']);
     this.setRenderingApplicationDimension(dimensions['rendering-application']);
-    this.setAbTestDimension(dimensions['ab-test']);
   };
 
+  StaticAnalytics.prototype.setAbTestDimensionsFromMetaTags = function() {
+      var $abMetas = $('meta[name^="govuk:ab-test"]'),
+          staticAnalytics = this,
+          // This is the block of dimensions assigned to A/B tests
+          minAbTestDimension = 40,
+          maxAbTestDimension = 49;
+
+      $abMetas.each(function() {
+        var $meta = $(this),
+        dimension = parseInt($meta.data('analytics-dimension')),
+        testNameAndBucket = $meta.attr('content');
+
+        if (dimension >= minAbTestDimension && dimension <= maxAbTestDimension) {
```

Comment:
Is it OK to silently ignore invalid or out-of-range dimensions? I'd prefer to throw an error here to alert the developer to the bad configuration, but I don't want people to accidentally break all analytics and not notice.